clients/src/main/resources/common/message/ShareAcknowledgeRequest.json

```diff
@@ -25,6 +25,8 @@
   // explicitly enabled.
   "latestVersionUnstable": true,
   "fields": [
+    { "name": "GroupId", "type": "string", "versions": "0+", "nullableVersions": "0+", "default": "null", "entityType": "groupId",
+      "about": "null if not provided or if it didn't change since the last fetch; the group identifier otherwise." },
```

Comment:
The KIP has been changed to provide the GroupId one every ShareFetch and ShareAcknowledge so that it's available as part of the key. So, the "about" should just say "The group identifier.".